core/src/main/java/io/grpc/internal/ClientStream.java

```diff
@@ -58,4 +58,14 @@
    * the remote end-point is closed. This method may only be called once.
    */
   void halfClose();
+
+  /**
+   * Starts stream. This method may only be called once.  It is safe to do latent initialization of
+   * the stream up until start is called.
```

Comment:
s/start/`{@link #start}`

Are other methods allowed to be called before `start()`?